app/views/admin/trends/statuses/index.html.haml

```diff
@@ -12,10 +12,11 @@
   .filters
     .filter-subset.filter-subset--with-select
       %strong= t('admin.follow_recommendations.language')
-      .input.select.optional
+      .input.select.select--languages.optional
         = form.select :locale,
                       options_for_select(@locales.map { |key| [standard_locale_name(key), key] }, params[:locale]),
-                      include_blank: true
+                      { include_blank: true },
+                      { disabled: @locales.empty? }
```

Comment:
Whilst doing the change to lift follow recommendations into trends, I noticed that the two language selects between "Follow Recommendations" and "Trending Posts" had varying widths, likely because of there being no options in "Trending Posts" unless trends have been calculated.

Before:
<img width="867" alt="Screenshot 2024-08-04 at 23 54 03" src="https://github.com/user-attachments/assets/d4097b9e-ac80-45ec-9551-88309be36640">

After:
<img width="923" alt="Screenshot 2024-08-04 at 23 53 23" src="https://github.com/user-attachments/assets/3ce1dbbd-ee62-4a33-bf74-c7818c7e47fc">

Reply:
This also disables the select if there's no options, which feels "right" though I'm not 100% sure.